code/replication/object/Similarity.py

```diff
@@ -0,0 +1,197 @@
+"""
+Class that can compare two different Articles. 
+"""
+
+from collections import Counter
+import numpy as np
+from nltk.corpus import stopwords 
+from nltk.tokenize import word_tokenize
+import nltk
+import heapq
+from nltk.stem.porter import *
+
+
+class Similarity:
+
+	def __init__(self, num_closest = 5, old_news = 0.6, reprint = 0.8, look_back_days = 5):
+		self.old_news = old_news
+		self.reprint = reprint
+		self.num_closest = num_closest
+		self.look_back = 51840 * look_back_days
```

Comment:
51,840 == 14.4 hours, which multiplied by the default argument of 5 days == 72 hours. I don't get why the look_back_days parameter is being scaled by 51840 seconds. Shouldn't the look back days be preserved by multiplying by 86,400 seconds? Whatever the answer, there should be a clarifying comment for the magic number. As it is, the look_back_days parameter is misleading. How are we accounting for the nuances of holidays/weekends? The lookback should act like weekends and market holidays don't exist... i.e. a story coming out 3pm on Tuesday can be compared to an story coming out since the prior Tuesday at 3pm (assuming a lookback period of 5 days).